dtale/exposed_ui/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM python:3.7
```

This does not build correctly on my machine, I bumped it to `python:3.10` and it builds without issues now. Was there any specific reason to leave it to `3.7`?

```suggestion
FROM python:3.10
```